Improve docs: publishPlanResults should not be the name of a subfolder of workingDirectory

**Describe the bug**
When publishPlanResults is the name of a subfolder of workingDirectory, say 'SomeName', the task will fail with error:

##[error]Error: EISDIR: illegal operation on a directory, open '/home/vsts/work/1/s/iac/terraform/SomeName'

**To Reproduce**
Steps to reproduce the behavior:
1. Set publishPlanResults as the same name of some subfolder of where you will run the terraform task.
2. Execute pipeline
3. See error

**Expected behavior**
At a minimum, docs contain a note about not using a subfolder name as the publishPlanResults name.
Ideally, the error is caught and proper guidance is provided to the user, or the task can work around this limitation.

**Agent Configuration**
 - Hosted
 - Terraform v0.15.3 on linux_amd64
 - Task version used 0.6.23

**Additional context**

--- a/overview.md
+++ b/overview.md
@@ -232,7 +232,7 @@ Use output variables as pipeline variables
 
 ## Terraform Plan View
 
-The extension includes a feature to render terraform plans within the pipeline run summary. To use this feature `publishPlanResults` input must be provided when running `terraform plan` via the `TerraformCLI` task. Then input should be set to the name that should be assigned to the plan.
+The extension includes a feature to render terraform plans within the pipeline run summary. To use this feature the `publishPlanResults` input must be provided when running `terraform plan` via the `TerraformCLI` task. The input should be set to the name that should be assigned to the plan (however, do not use the name of a subfolder in the working directory).
 
 > **Important** - When enabling `publishPlanResults` the `-detailed-exitcode` option will be added when running terraform plan if it was not already provided in the `commandOptions` input. This will cause `TERRAFORM_LAST_EXITCODE` to be `2` when plan includes changes; which is a successful exitcode. Additionally, warnings will be logged to the pipeline summary when changes are present as a means to alert that changes will be made if the templates are applied.
 

